src/components/OSCALLoader.js

```diff
@@ -44,7 +44,7 @@ const EditorSplit = styled(Split)`
 export default function OSCALLoader(props) {
   const [isLoaded, setIsLoaded] = useState(false);
   const [isResolutionComplete, setIsResolutionComplete] = useState(false);
-  const { isRestMode, setIsRestMode } = props;
+  const { isRestMode, backendUrl } = props;
```

Comment:
Can we just not do this and always refer to them as `props.isRestMode` and `props.backendUrl`? I think when it's like this, it's hard to tell that these get passed in which I think is helpful throughout the function here.

We don't also define `oscalUrl`, `oscalObjectType`, etc... we rely on accessing those via `props`. So let's be consistent.

Reply:
Okay, yeah it would probably make sense to make this `props.isRestMode` and `props.backendUrl`. I was just trying to accommodate for the removal of `setIsRestMode`.